lib/miq_automation_engine/engine/drb_remote_invoker.rb

```diff
@@ -60,10 +64,15 @@ def drb_cache_timeout
       1.hour
     end
 
+    def api_token
+      tm = TokenManager.new("api")
+      tm.gen_token("api", :userid => @workspace.ae_user.userid)
```

Comment:
probably related to https://github.com/ManageIQ/manageiq/pull/10808 where we were getting ready for websocket support.